common/src/test/java/org/apache/sedona/common/raster/RasterAccessorsTest.java

```diff
@@ -201,6 +201,23 @@ public void testGridCoordXLonLat() throws FactoryException, TransformException {
         assertEquals(expectedX, actualX);
     }
 
+    @Test
+    public void testSkewX() throws FactoryException {
+        GridCoverage2D emptyRaster = RasterConstructors.makeEmptyRaster(10, 2, 4, 6, 4, 1, 1, 2, 7, 0);
+        assertEquals(2, RasterAccessors.getSkewX(emptyRaster), 0.1d);
```

Comment:
Please add tests testing the convenience constructor of MakeEmptyRaster with 0 skews

Reply:
Okay